docs/dashboard/x-ray.md

```diff
@@ -49,38 +49,68 @@ http.clientip = 1.2.3.4
+The flamegraph tab provides you with a full performance profile of the request, helping you to understand how the request was processed and executed.
 
-The Flamegraph is a graph showing the request time on the X axis, and the call stack / depth on the Y axis. A tall Flamegraph segment represents a deep call-stack (not necessarily an indication of a problem) and a wide segment represents a slow operation (which can indicate a performance bottleneck).
+The flamegraph is a graph showing the request time on the X axis, and the call stack / depth on the Y axis. Each entry in the call-stack forms a "segment", and you can hover over any segment to view more details or click it to zoom to just that segment. (Click on the top segment to zoom back out.)
 
-Flamegraphs are taken from sampled profiles of the PHP process, at 5 millisecond intervals. This means the resolution of the whole Flamegraph is 5ms. Sampled profiles means there is negligible performance overhead to collecting such profiles.
+Flamegraphs are taken from sampled profiles of the PHP process, at 5 millisecond intervals. Tall segments indicate a deep call-stack, but do not necessarily indicate a problem, while wide segments indicate slower operations. When investigating performance problems, start with the wide segments to understand what is taking a long time to run, and use the deeper segments to drill down and understand the operations taking place.
+
+Note that as the resolution of the whole flamegraph is 5ms, all times are rounded. Fast functions may still appear in the flamegraph if they happened to be running when the sample was taken, and for this reason, flamegraphs are more valuable for slower requests. (A 5ms interval is used as the sampling rate to ensure negligible performance overhead when collecting the profiles.)
 
 ### Request Data
 
-Each Xray trace includes all of the request data, which can aid in debugging unexpected behaviour. The request data includes all `$_POST`, `$_GET` and `$_COOKIE`, `$_SERVER` values, and information for the HTTP response such as response code and all sent HTTP headers.
+![Example request data](../assets/xray-request.png)
+
+The request tab displays data sent to and served from the server, including the client IP, request and response headers, and any other statistics generated by the request. This can assist in debugging unexpected behaviour by replicating the conditions, or finding unexpected header values.
+
+Some information is redacted from the request data, for privacy reasons, including user passwords. You can redact additional information by filtering `aws_xray.redact_metadata_keys` or `aws_xray.redact_metadata`:
+
+```php
+// Redact keys from the metadata:
+add_filter( 'aws_xray.redact_metadata_keys', function ( $keys ) {
+    $keys['$_POST'][] = 'rcp_user_pass';
```

Comment:
This is a bit hard to understand, more so with the removal of what the metadata keys are e.g. `$_POST`, `$_GET` and `$_COOKIE`, `$_SERVER`.

When would I use redact metadata over redact metadata keys?

Reply:
I asked myself the same question; I think basically if you don't know the keys ahead of time

Reply:
Right but we know some basics, I think this PR would be a good opportunity to expand on these filter docs but we can also add it later.

Reply:
@rmccue do you have time to make these updates or do you need me to complete it?